sockeye/training.py

```diff
@@ -305,16 +305,17 @@ def save_params_to_file(self, fname: str):
         self.aux_params = aux_params
         super().save_params_to_file(fname)
 
-    def load_params_from_file(self, fname: str, set_params: bool = True):
+    def load_params_from_file(self, fname: str, allow_missing_params: bool = False):
         """
         Loads parameters from a file and sets the parameters of the underlying module and this model instance.
 
         :param fname: File name to load parameters from.
         :param set_params: Set the params in the module.
```

Comment:
Update doc string:
":param allow_missing_params: If True, the parameters in fname are allowed to be a subset of the Module parameters."